README.md

```diff
@@ -1,107 +1,65 @@
-```
-
-The platform/architecture type of the package created will match the platform
-where the `build project` command is invoked. So running this command on say a
-MacBook Pro will generate a Mac OS X specific package. After the build
-completes packages will be available in `pkg/`.
-
-### Clean
+Grab Vagrant from http://vagrantup.com and follow the instructions.
 
-You can clean up all temporary files generated during the build process with
-the `clean` command:
 
 ```shell
-$ bin/omnibus clean
+$ vagrant plugin install vagrant-omnibus vagrant-vbguest vagrant-berkshelf
 ```
 
-Adding the `--purge` purge option removes __ALL__ files generated during the
-build including the project install directory (`/opt/nginx-openresty`) and
-the package cache directory (`/var/cache/omnibus/pkg`):
-
 ```shell
-$ bin/omnibus clean --purge
+$ sudo gem install bundler && sudo gem update bundler
 ```
 
-### Help
-
-Full help for the Omnibus command line interface can be accessed with the
-`help` command:
-
 ```shell
-$ bin/omnibus help
+$ bundle install --binstubs
 ```
 
-## Vagrant-based Virtualized Build Lab
-
-Every Omnibus project ships will a project-specific
-[Berksfile](http://berkshelf.com/) and [Vagrantfile](http://www.vagrantup.com/)
-that will allow you to build your projects on the following platforms:
-
-* CentOS 5 64-bit
-* CentOS 6 64-bit
-* Ubuntu 10.04 64-bit
-* Ubuntu 11.04 64-bit
-* Ubuntu 12.04 64-bit
+## Usage
 
-Please note this build-lab is only meant to get you up and running quickly;
-there's nothing inherent in Omnibus that restricts you to just building CentOS
-or Ubuntu packages. See the Vagrantfile to add new platforms to your build lab.
+### Build
 
-The only requirements for standing up this virtualized build lab are:
+Kitchen-based Build Environment
+-------------------------------
 
-* VirtualBox - native packages exist for most platforms and can be downloaded
-from the [VirtualBox downloads page](https://www.virtualbox.org/wiki/Downloads).
-* Vagrant 1.2.1+ - native packages exist for most platforms and can be downloaded
-from the [Vagrant downloads page](http://downloads.vagrantup.com/).
+Decide which ubuntu release you want to build for.
 
-The [vagrant-berkshelf](https://github.com/RiotGames/vagrant-berkshelf) and
-[vagrant-omnibus](https://github.com/schisamo/vagrant-omnibus) Vagrant plugins
-are also required and can be installed easily with the following commands:
+`export RELEASE=ubuntu-1204` or `export RELEASE=ubuntu-1404`
 
 ```shell
-$ vagrant plugin install vagrant-berkshelf
-$ vagrant plugin install vagrant-omnibus
+bundle exec kitchen converge openresty-$RELEASE
 ```
 
-Once the pre-requisites are installed you can build your package across all
-platforms with the following command:
+Then login to the instance and build the project:
 
 ```shell
-$ vagrant up
-```
+bundle exec kitchen login openresty-$RELEASE
+...
+cd omnibus-nginx-openresty
+bundle install --without development # Don't install dev tools!
```

Comment:
This comment doesn't give any more info then the command itself. Remove it or make it say why we don't need the dev tools?